src/python/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ endfunction( add_gudhi_debug_info )
 if(PYTHONINTERP_FOUND)
   if(PYBIND11_FOUND)
     add_gudhi_debug_info("Pybind11 version ${PYBIND11_VERSION}")
+    set(GUDHI_PYTHON_MODULES "${GUDHI_PYTHON_MODULES}'bottleneck', ")
```

Comment:
Does it imply that `gudhi.bottleneck` won't be available without Pybind11 ? (I don't say its bad, just being curious).

Reply:
Yes, if you don't have pybind11 at build time, bottleneck won't be built (it is the same with cython now). But actually, pybind11 is already a mandatory build dependency, gudhi refuses to build the python plugin at all if you don't have it, IIRC (I could misremember).
You don't need it at runtime though.